src/updatees/travis.js

```diff
@@ -3,24 +3,40 @@ const fs = require('fs');
 const c = require('chalk');
 const _ = require('lodash/fp');
 const Promise = require('bluebird');
-const yaml = require('js-yaml');
+const YAML = require('yaml');
 
 const readFile = Promise.promisify(fs.readFile);
 const writeFile = Promise.promisify(fs.writeFile);
 
-const updateTravis = (node, travis) => {
-  if (_.isArray(travis)) return Promise.map(travis, t => updateTravis(node, t));
-
-  if (!travis || !node) return Promise.resolve();
+const patchVersionInTravisYaml = nodeVersion => yamlString => {
+  const yamlCst = YAML.parseCST(yamlString);
+  const documentItems = yamlCst[0].contents[0].items;
```

Comment:
are you sure that the following indexes will be available ? imo it could be kind to check that the value is defined before using it

Reply:
yep: first `cst[0]`is about getting the first document. (travis.yml have just one), then contents is about the statement.

Reply:
but indeed, better make this resilient